registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidator.java

```diff
@@ -526,8 +573,24 @@ private boolean validateIntroducerBiometric(String registrationId,
 			InternalRegistrationStatusDto registrationStatusDto,
 			String introducerUIN) throws IOException, ApisResourceAccessException,
 			BioTypeException, AuthSystemException, JsonProcessingException, PacketManagerException, CertificateException, NoSuchAlgorithmException {
+		String process=registrationStatusDto.getRegistrationType();
+		
+		if(!subProcesses.contains(process)) {
+		InfoResponseDto infoResponseDto=packetManagerService.getInfo(registrationId);
```

Comment:
same here. Should not call packetManagerService.getInfo(registrationId); 
instead u should call packetManagerService.getMetaInfo method.